docs/style.md

```diff
@@ -0,0 +1,65 @@
+# Style guidelines
+
+## Technical requirements
+- All files **must** be encoded using UTF-8 (without BOM)
+- Use only Unix line endings (`\n`)
+
+## Line lengths
+Please aim to keep lines in an XML file around 80 characters long or less.
+This is a loose requirement and 100 is probably acceptable as a maximum length.
+This aids in keeping *diffs* simple, which is particularly useful for translators,
+so follow this rule carefully.
+
+## Whitespaces
+For XML, indent using one space. Do not use tabs. PHP code, in examples, uses
+four spaces for indentation, since the code should adhere to the [PEAR Coding Standards](http://pear.php.net/manual/en/standards.php).
+
+## Punctuation
+Punctuation in the PHP Manual follows regular grammatical rules. When writing flowing sentences, such as in function
+descriptions, normal punctuation should be used. Lists, titles, and sentence fragments should not be punctuated with
+a period. Sentences need not have two spaces between them. Commas and apostrophes should be used appropriately.
+
+## Personalization
```

Comment:
It might be useful to separate this page into a general and an English style guide as everything above this line applies to all languages/translations while more or less everything below it applies to `doc-en` only.